src/test/scala/uk/gov/hmrc/play/auth/microservice/filters/AuthorisationFilterSpec.scala

```diff
@@ -33,7 +35,9 @@ import scala.concurrent.Future
 
 class AuthorisationFilterSpec extends WordSpecLike with MatchersResults with ScalaFutures {
 
- 
+  implicit val system = ActorSystem()
+  implicit val materializer = ActorMaterializer()
```

Comment:
Play have moved form iteratees to akka streams in a number of places in 2.5 (see https://www.playframework.com/documentation/2.5.x/Highlights25#New-streaming-API-based-on-Akka-Streams) including filters.
Theses require an implicit materializer (which also requires an actor system), generally for play this will be provided by DI but to smooth the upgrade path to 2.5 we are avoiding using full DI in mdtp for the time being.
Any concrete instances of the AuthorisationFilter filter require an implicit to be declared - hence the declarations in the Spec, it will also be required in the objects which extend this filter although for most microservices this will be covered by changing the *-bootstrap libraries.